network-api/networkapi/settings.py

```diff
@@ -33,6 +33,7 @@
     CORS_WHITELIST=(tuple, ()),
     DATABASE_URL=(str, None),
     DEBUG=(bool, False),
+    DISABLE_DEBUG_TOOLBAR=(bool, False),
```

Comment:
Not a blocker, but I wonder if we're approaching a point where we should have an "environment" variable that sets a few of these automatically (set to "development" or "production")

Reply:
That would be `DEBUG`. if you turn that on and don't set anything else, the debug toolbar will work just fine. This env var is for explicitly *disabling* the debug toolbar, despite DEBUG being true (for instance, it's getting in the way of some particular design/testing work)